cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,162 @@
+    cy.get('#salary').type(fakeUser.salary);
+    cy.get('#department').type(fakeUser.department);
+    cy.get('#submit').click();
+    cy.get(':nth-child(4) > .rt-tr > :nth-child(1)').should('contain', `${fakeUser.firstName}`);
+  });
+
+  it('Delete a worker', () => {
+    cy.get('#delete-record-1 > svg').click();
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(1)')
+      .should('not.contain', workers[0].firstName);
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(2)')
+      .should('not.contain', workers[0].lastName);
+  });
+
+  it('Finds and edits a worker, check the data after editing', () => {
+    cy.get('#searchBox').type(workers[0].firstName);
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(1)')
+      .should('contain', workers[0].firstName);
+    cy.get('#edit-record-1').click();
+    cy.get('#firstName').clear().type('Anastasiia');
+    cy.get('#lastName').clear().type('Holland');
+    cy.get('#userEmail').clear().type('tomholland@gmail.com');
+    cy.get('#submit').click();
+    cy.get('#searchBox').clear();
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(1)')
+      .should('contain', 'Anastasiia');
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(2)')
+      .should('contain', 'Holland');
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(4)')
+      .should('contain', 'tomholland@gmail.com');
+  });
+
+  it('Checks search by all column values', () => {
+    cy.get('#searchBox').type(workers[1].firstName);
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(1)')
+      .should('contain', workers[1].firstName);
+    cy.get('#searchBox').clear().type(workers[2].lastName);
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(2)')
+      .should('contain', workers[2].lastName);
+    cy.get('#searchBox').clear().type(workers[0].age);
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(3)')
+      .should('contain', workers[0].age);
+    cy.get('#searchBox').clear().type(workers[0].email);
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(4)')
+      .should('contain', workers[0].email);
+    cy.get('#searchBox').clear().type(workers[2].salary);
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(5)')
+      .should('contain', workers[2].salary);
+    cy.get('#searchBox').clear().type(workers[1].department);
+    cy.get('.rt-tbody > :nth-child(1) > .rt-tr > :nth-child(6)')
+      .should('contain', workers[1].department);
+  });
+
+  it('Checking pagination', () => {
+    cy.get('.-pagination').should('exist');
+    cy.get('.-center').should('exist');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+    cy.get('.-pageJump').should('exist');
+
+    addNewUser(fakeUser);
+    addNewUser(fakeUser);
+    addNewUser(fakeUser);
+
+    cy.get('select').should('exist');
+    cy.get('select').select('5');
+    cy.get('.rt-tbody .rt-tr-group').should('have.length', 5);
```

Comment:
```suggestion
    cy.get('select').should('exist');
    cy.get('select').select('5');
    cy.get('.rt-tbody .rt-tr-group').should('have.length', 5);
```
you already tested it in the first test